File: java-main/java-main/IO/Nestedfor5/p10.java
/*write a program to print a series of prime numbers from entered range.*/


import java.util.Scanner;
class Demo{
    public static boolean isPrime(int num) {
        if (num <= 1) {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(num); i++) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter starting number: ");
        int start = input.nextInt();
        System.out.print("Enter ending number: ");
        int end = input.nextInt();
        input.close();
        System.out.print("Series = ");
        for (int i = start; i <= end; i++) {
            if (isPrime(i)) {
                System.out.print(i + " ");
            }
        }
    }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session3/p10.c
/* 10
   I H
   7 6 5 
   D C B A*/
#include<stdio.h>
void main(){

	int rows;
	
	

	printf("Enter no of rows:\n");
	scanf("%d",&rows);

	int num;

	for(int i=1;i<=rows;i++){

		for(int j=1;j<=i;j++){

			printf("%d ",num);
			num--;
		}
		printf("\n");
	}
}


